content/vscode/remote.md

```diff
@@ -0,0 +1,134 @@
+---
+title: มา Dev กันแบบไม่ต้องลง SDK กันเถอะ (Developing inside a Container) 
+showMetadata: true
+editable: true
+showToc: true
+---
+
+
+
```

Comment:
Very minor, in Markdown these there empty lines will get the same result as one empty line.
Therefore, please remove two empty line and keep one empty line.